codespell_lib/data/dictionary.txt

```diff
@@ -17698,6 +17703,8 @@ seleted->selected
 self-contianed->self-contained
 self-referencial->self-referential
 self-refering->self-referring
+selt->set
+selted->set
```

Comment:
Suggest settled

Reply:
Suggest setted first.

Reply:
`setted` isn't a valid word

Reply:
Oxford Dictionary seems to disagree (although likely rare): https://www.lexico.com/en/definition/setted

Possibly also settee.

Reply:
My main objection is this encourages auto-correction of things that may not be correct, you're expecting both a typo AND a spelling mistake. Salted, melted, belted, pelted, felted seem more likely to me, especially the former.

Could we compromise on "disabled due to being unlikely", which I think will still throw it up but not auto-correct?

Reply:
I've been thinking about it and since we're running in to usecases where the word exists but is a rarity, lets consider the idea that was brought up of a secondary dictionary that can be linked to test against ? `setted` and friends can be in it ?